lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -353,8 +353,8 @@ def test_entrance_exam_requirement_message_hidden(self):
         )
         resp = self.client.get(url)
         self.assertEqual(resp.status_code, 200)
```

Comment:
no need for checking status_code, assertNotContains checks that.

Reply:
this isn't the part of the change, do we want to refactor this?